docs/draft/howto/catalog-queries-graphql-endpoint.md

```
@@ -0,0 +1,212 @@
+
+You also need to port forward the catalog server service:
+
+``` terminal
+kubectl -n olmv1-system port-forward svc/catalogd-service 8443:443
+```
+
+## GraphQL Endpoint
+
+The GraphQL endpoint is available at:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)